### PR TITLE
[TiDB] Add new mode option `prefer-leader` for `tidb_replica_read`.

### PR DESCRIPTION
**Pull request by LykxSassinator**
Signed-off-by: Lucasliang <nkcs_lykx@hotmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a new mode `prefer-leader` for the system variable `tidb_replica_read`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/pull/12337

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-use-follower-read.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ aliases: ['/zh/tidb/dev/use-follower-read']
 <SimpleTab>
 <div label="SQL">
 
-在 SQL 中，你可以将变量 `tidb_replica_read` 的值（默认为 `leader`）设置为 `follower`、 `leader-and-follower`、 `closest-replicas` 或 `closest-adaptive` 开启 TiDB 的 Follower Read 功能：
+在 SQL 中，你可以将变量 `tidb_replica_read` 的值（默认为 `leader`）设置为 `follower`、 `leader-and-follower`、 `prefer-leader`、 `closest-replicas`、 `closest-adaptive` 或 `prefer-leader` 开启 TiDB 的 Follower Read 功能：
```
Comment (TomShawn) [inline develop/dev-guide-use-follower-read.md:36]:
```suggestion
在 SQL 中，你可以将变量 `tidb_replica_read` 的值（默认为 `leader`）设置为 `follower`、`leader-and-follower`、`prefer-leader`、`closest-replicas`、`closest-adaptive` 或 `prefer-leader` 开启 TiDB 的 Follower Read 功能：
```

File: follower-read.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ set [session | global] tidb_replica_read = '<目标值>';
 - 当设置为默认值 `leader` 或者空字符串时，TiDB 会维持原有行为方式，将所有的读取操作都发送给 leader 副本处理。
 - 当设置为 `follower` 时，TiDB 会选择 Region 的 follower 副本完成所有的数据读取操作。
 - 当设置为 `leader-and-follower` 时，TiDB 可以选择任意副本来执行读取操作，此时读请求会在 leader 和 follower 之间负载均衡。
+- 当设置为 `prefer-leader` 时，TiDB 会优先选择 leader 副本执行读取操作。当 leader 副本的处理速度明显变慢时，例如由于磁盘/网络性能抖动，TiDB 将选择其他可用的 follower 副本来执行读取操作。
```
Comment (TomShawn) [inline follower-read.md:38]:
```suggestion
- 当设置为 `prefer-leader` 时，TiDB 会优先选择 leader 副本执行读取操作。当 leader 副本的处理速度明显变慢时，例如由于磁盘或网络性能抖动，TiDB 将选择其他可用的 follower 副本来执行读取操作。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by LykxSassinator** [comment]
@TomShawn As all related PRs merged, this pr can be merged now.

**Comment by TomShawn** [comment]
> 

This PR needs approval by @easonn7.

**Review by easonn7 — APPROVED** [review]
LGTM

**Pull request merged**